sormas-e2e-tests/src/main/java/org/sormas/e2etests/enums/APITestData/Disease.java

```diff
@@ -0,0 +1,60 @@
+package org.sormas.e2etests.enums.APITestData;
+
+public enum Disease {
```

Comment:
Diseases*

Reply:
It holds the disease type, and I think it sounds better Disease.CORONAVIRUS than Diseases.CORONAVIRUS.

Reply:
in terms of reading, could be, but in terms of logic, the enum is used as a list, and all lists have the plural. It's a minor thing, I won't insist so if you want we can keep it like this